src/workloads/docs/Generators.yml

```diff
@@ -251,6 +251,22 @@ Actors:
             # The following example returns - repeatingSeq: [1, 2, 3, 1, 2]
             repeatingSeq: {^Array: {of: {^Choose: {from: [1, 2, 3], deterministic: true}}, number: 5}}
 
+            # You can also filter items out, similar to a set, by setting `distinct` = true.
+            # For example, you can generate N distinct random integers.
+            distinctSeq10Ints: {^Array: {of: {^Repeat: {count: 10, fromGenerator: {
+              ^Inc: {start: 0, end: 1000}}}}, number: 10, distinct: true}}
+
+            # Just be careful, the generator will give up if the failure rate gets too high, such as
+            # with this next generator.
+            distinctSeqFailure: {^Array: {of: {^Repeat: {count: 10000, fromGenerator: {
+              ^Inc: {start: 0, end: 1000}}}}, number: 10, distinct: true}}
+
+            # A good rule of thumb is that you should be selecting less than 50% of your range.
+            # You're not limited to integers, though! You can also filter doubles, strings, even
+            # documents and arrays.
+            distinctSeq10Strings: {^Array: {of: {^Repeat: {count: 10, fromGenerator: {
```

Comment:
I would also take the repeat out of here, since it inhibits the copy/paste-ability.